script.js

```diff
@@ -9,6 +9,17 @@ let nameInfo = () => {
   var button = document.getElementById('btn');
   button.addEventListener('click',nameInfo);
 
+let personalInfo = () => {
```

Comment:
Why use `let` ?

Reply:
I thought on arrow functions, they needed to be defined first.. that is how I've done it in the past.